exercises/B-hello-world/exercise.js

```diff
@@ -1 +1,6 @@
 console.log("Hello world");
+console.log("I started to learn javascript!");
+console.log("Good Evening");
+console.log("I am thrilled!");
+//console.log(I started to learn javascript!);/* I got error message for not using quote marks.*/
```

Comment:
It is definitely worth getting used to what error messages look like in JavaScript and try to interpret what they mean. You'll get plenty of practice through the course! In this case you will get a Syntax Error, which means that the JavaScript isn't valid.